### ISSUE TITLE
test failure for eccodes v2.34.1

### ISSUE THREAD

**Issue by jdkloe** [jswhit/pygrib#245]
After updating eccodes from v2.33.0 to v2.34.1 one of the test cases in the test suite fails. I get this output:
```
[snip]
**********************************************************************
File "/builddir/build/BUILD/pygrib-2.1.5-tmp-test/test/test.py", line 237, in __main__.test
Failed example:
    for grb in grbs: print(grb)
Expected:
    1:Total precipitation:kg m-2 (accum):lambert:surface:level 0:fcst time 15-30 mins (accum):from 201804100000
Got:
    1:Total precipitation:kg m-2 (accum):lambert:surface:level 0:fcst time 15m-30m mins (accum):from 201804100000
[snip]
**********************************************************************
1 items had failures:
   1 of 110 in __main__.test
110 tests in 2 items.
109 passed and 1 failed.
***Test Failed*** 1 failures.
using ECCODES library version 2.34.1
```
So it looks  like eccodes has changed its formatting for this specific type of output strings.

**Comment by jswhit** [comment]
Should be fixed by PR #254

**Comment by jdkloe** [comment]
Thanks for this fix, I can confirm that the issue is solved now.
